src/main/java/com/dev/cinema/exception/DataPrcessingException.java

```diff
@@ -0,0 +1,7 @@
+package com.dev.cinema.exception;
+
+public class DataPrcessingException extends Exception {
```

Comment:
this exception is created, but never used. remove it or use in some method

Reply:
I created for future. I will delete